ci-operator/step-registry/gather/ibmcloud-resource/gather-ibmcloud-resource-commands.sh

```diff
@@ -138,13 +138,22 @@ function gather_cis {
 
 # Gather resources
 function gather_resources {
+    local hasSetTarget=false
     for resource in "${MAIN_RESOURCES[@]}"; do
         {
             echo -e "# ibmcloud is ${resource}s\n"
+            if [[ ${resource} == "image" ]] && [[ ! -z $RESOURCE_GROUP ]]; then
```

Comment:
if `RESOURCE_GROUP` is empty, your code would never can be executed to here, the line 169 would directly exit the whole step, that means `[[ ! -z $RESOURCE_GROUP ]]` become meaningless.

Reply:
ok